app/services/weet.js

```diff
@@ -0,0 +1,11 @@
+const { frikiPhraseRepository } = require('../repositories/witter/friki_phrase');
+
+const getRandomWeet = async phraseRepository => {
+  try {
+    return await phraseRepository.getPhrase();
+  } catch ({ message }) {
+    throw Error(message);
+  }
+};
+
+exports.getRandomWeet = () => getRandomWeet(frikiPhraseRepository);
```

Comment:
This is fine, but what is the purpose of passing the repository like this?

Reply:
When using repository I wanted to apply the open/closed principle in case I have another repository where I don't get the data through axios but another sdk.

Or do you mean there is another way to instantiate it?

Reply:
No, I just wanted to know the reasoning behind this 👌